mempool/metrics.go

```diff
@@ -37,8 +38,16 @@ type Metrics struct {
 	// CheckTx.
 	EvictedTxs metrics.Counter
 
+	// The amount of transactions that successfully make it into a block.
```

Comment:
```suggestion
	// SuccessfulTxs defines the number of transactions that successfully made
	// it into a block.
```

Proposal to use "count" or "number" instead of "amount" because we can individually count the number of successful transactions. The suggestion uses "number" to conform to the comment above for `FailedTxs`.

Motivation:

> By contrast, amount refers to how much of something there is: how much stimulation, how much resistance, or how much liquid. In effect, amount refers to quantities we can measure but not individually count.

[source](https://crosstalk.cell.com/blog/the-difference-between-number-and-amount#:~:text=Number%20refers%20to%20how%20many,resistance%2C%20or%20how%20much%20liquid.)